app/src/main/res/layout/audio_fragment.xml

```diff
@@ -71,10 +75,7 @@
       android:paddingTop="16dp"
       android:paddingEnd="16dp"
       android:paddingBottom="16dp"
-      android:text="@{viewModel.currentLanguageCode}"
```

Comment:
Remove `currentLanguageCode` from ViewModel and its related code if not needed or make it private.

Reply:
@rt4914 I  made the `viewModel.currentLanguageCode` as content description and then i check the content description of the image in test in audioFragmentTest

Reply:
Yeah, I forgot to remove this comment when I saw the tests. Thanks.